jetty-webapp/src/test/java/org/eclipse/jetty/webapp/HugeResourceTest.java

```diff
@@ -359,6 +411,21 @@ protected void doPost(HttpServletRequest req, HttpServletResponse resp) throws I
         }
     }
 
+    public static class ChunkedServlet extends HttpServlet
+    {
+        @Override
+        protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws IOException
+        {
+            URL resource = req.getServletContext().getResource(req.getPathInfo());
+            OutputStream output = resp.getOutputStream();
+            try (InputStream input = resource.openStream())
+            {
+                resp.setContentType("application/octet-stream");
+                IO.copy(input, output);
```

Comment:
If you wanted to ensure chunking is used regardless of the relative size of the content to the buffer:

```suggestion
                resp.setContentType("application/octet-stream");
                resp.flushBuffers();
                IO.copy(input, output);
```

Reply:
Changed.